### ISSUE TITLE
Issues with ProtoBuf plugin and Julia 1.2

### ISSUE THREAD

**Issue by zhouyan** [JuliaIO/ProtoBuf.jl#132]
After a recent upgrade to Julia 1.2, I am running into the following errors when trying to generate source from protobuf

```
--julia_out: protoc-gen-julia: Plugin output is unparseable: Activating environment at `~/.julia/packages/ProtoBuf/W3nSD/Project.toml`\nz\260\002\n\017timestamp_pb.jlz\234\002# syntax: proto3\nusing ProtoBuf\n....
```

I omitted the rest of the error message which is just the contents of the file to be generated.

The issue appears to be the output from he `Pkg.activate` statement found in the `protoc-gen-julia` script, more specifically the line,

```
JULIA_GEN="${JULIA} ${COVERAGE} -e 'if VERSION >= v\"0.7-\"; import Pkg; Pkg.activate(joinpath(ENV[\"SCRIPT_DIR\"], \"..\")); end; using ProtoBuf; using ProtoBuf.Gen; gen()'"
```

Remove the activate part, that is shorten it to

```
JULIA_GEN="${JULIA} ${COVERAGE} -e 'using ProtoBuf; using ProtoBuf.Gen; gen()'"
```

Appears to solve the problem.

I can see that this is not a real solution as the purpose of the `activate` is to avoid possible issues of user's own environment. In my case there's none but that doesn't mean it is the general case.

Anyway, I don't think this is a bug of the package, we may need to find a workaround though.

At the core of the issues is that pre-1.2, the message is sent to STDERR, for example,

with Julia 1.1,

```
 julia -e 'using Pkg; Pkg.activate("~/.julia/packages/ProtoBuf/W3nSD/Project.toml")' | wc
[ Info: activating new environment at /opt/fairtide/tools/julia/share/julia/packages/ProtoBuf/W3nSD/Project.toml.
      0       0       0
```

and with Julia 1.2 its now send to STDOUT and thus captured as part of the plugin output

```
julia -e 'using Pkg; Pkg.activate("~/.julia/packages/ProtoBuf/W3nSD/Project.toml")' | wc
       1       5     121
```

A possible workaround is:

```
JULIA_GEN="${JULIA} ${COVERAGE} -e 'if VERSION >= v\"0.7-\"; import Pkg; redirect_stdout(() -> Pkg.activate(joinpath(ENV[\"SCRIPT_DIR\"], \"..\")), stderr); end; using ProtoBuf; using ProtoBuf.Gen; gen()'"
```

that is, redirect the stdout of the activate statement. This shall work for both Julia 1.2 and older versions. It has no real effect in the later case.

**Comment by zhouyan** [comment]
I see there's already a PR for this issue #129

**Comment by tanmaykm** [comment]
Thanks. Should be fixed by #129.
